curl.txt

```diff
@@ -1,11 +1,11 @@
-[Section]
-Name = cURL
-Version = 7.80.0
-Category = 12
-URLDownload = https://web.archive.org/web/20230529150511if_/https://curl.se/windows/dl-7.80.0_2/curl-7.80.0_2-win32-mingw.zip
-SizeBytes = 5132638
-
-Description = Command line tool and library for transferring data with URL syntax.
-License = The curl license
-SHA1 = e998537335c1735d0e214275964bbf2f6b6c9539
-URLSite = https://curl.se
+[Section]
+Name = cURL
+Version = 7.80.0
+Category = 12
+URLDownload = https://web.archive.org/web/20230529150511if_/https://curl.se/windows/dl-7.80.0_2/curl-7.80.0_2-win32-mingw.zip
+SizeBytes = 5132638
+
+Description = Command line tool and library for transferring data with URL syntax.
+License = The curl license
+SHA1 = e998537335c1735d0e214275964bbf2f6b6c9539
+URLSite = https://curl.se
```

Comment:
Diff is unclear (changed linebreaks) in this file and in some others, please fix.

Reply:
Same as all ffmpeg files, I commented that in the chat few days ago. Seems to be a format issue? Git always told me I made changes to this files but I didn't touch anything.

Reply:
> Same as all ffmpeg files, I commented that in the chat few days ago. Seems to be a format issue? Git always told me I made changes to this files but I didn't touch anything.

You probably have `core.autocrlf` enabled in this repo, or you changed that after cloning this repo.

Reply:
Your theory seems true. I am happy to know that this auto CR-LF thingy is enabled.